### PR TITLE
New target: SDMODELH7V1

### PR DESCRIPTION
**Pull request by sensei-hacker**
New Target that is very similar to the Kakute H7.

### CODE REVIEW

**Comment by sensei-hacker** [comment]
I have bench tested in both quad and airplane configurations and see no problems.
I tested buzzer, uart (GPS & rx), Dshot and pwm outputs.

**Comment by MrD-RC** [comment]
I've received a sample of this FC, as per the new hardware policy. Seems good to me. Will check off as I complete tests.

- [ ] Flash firmware
- [ ] Calibrate
- [ ] Orientation matches
- [ ] Gyro working
- [ ] Accel working
- [ ] Baro working
- [ ] Voltage correct
- [ ] Current correct
- [ ] UARTs working
- [ ] Camera working
- [ ] Video Out working
- [ ] OSD working
- [ ] LEDs working
- [ ] Buzzer working

File: src/main/target/SDMODELH7V1/target.c [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+#include "drivers/bus.h"
+#include "drivers/io.h"
+#include "drivers/pwm_mapping.h"
+#include "drivers/timer.h"
+#include "drivers/pinio.h"
+#include "drivers/sensor.h"
+
+timerHardware_t timerHardware[] = {
+
+    DEF_TIM(TIM3, CH3, PB0,     TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR, 0, 0),   // S1
```
Comment (mmosca) [inline src/main/target/SDMODELH7V1/target.c]: You can change all servo/motor outputs to TIM_USE_OUTPUT_AUTO :)

Comment (sensei-hacker) [reply src/main/target/SDMODELH7V1/target.c]: I could. But that would make it more cool.  So I did.

**Pull request merged**